src/components/sections/judge/JudgeRewardsPayoutTableRow.tsx

```diff
@@ -0,0 +1,59 @@
+import Link from 'next/link'
+
+import {Button} from '@/components/ui/Button'
+import {TableCell, TableRow} from '@/components/ui/Table'
+import {usePayReward} from '@/lib/queries/reward/payReward'
+import {formatAda, formatDate, formatTxHash} from '@/lib/utils/common/format'
+import {RewardsPayout} from '@/server/actions/reward/getReward'
+
+type JudgeRewardsPayoutTableRowProps = {
+  reward: RewardsPayout
+}
+
+const JudgeRewardsPayoutTableRow = ({
+  reward,
+}: JudgeRewardsPayoutTableRowProps) => {
+  const {mutate, isPending} = usePayReward()
+
+  return (
+    <TableRow>
+      <TableCell className="text-titleS">
+        {reward.user?.name ?? reward.user?.alias ?? '-'}
+      </TableCell>
+      <TableCell className="text-bodyM">{reward.user?.email ?? '-'}</TableCell>
+      <TableCell className="text-bodyM">
+        {reward.user?.walletAddress ?? '-'}
```

Comment:
The full address doesn't fit on laptop screens. Maybe it could be formatted with an ellipsis in the middle, and then some copy icon, which would copy it to the clipboard.